pkg/api/v1alpha1/cluster_types.go

```diff
@@ -268,6 +270,12 @@ type ClusterList struct {
 	Items           []Cluster `json:"items"`
 }
 
+// Version of scylla docker starting from which passing arguments via entry-point is supported
+var ScyllaVersionThatSupportsArgsText = "4.2.0"
+var ScyllaVersionThatSupportsArgs *semver.Version
```

Comment:
why pointer? Every usage dereference it. Use value.

Reply:
fixed